client/package.json

```diff
@@ -44,14 +44,16 @@
     "lodash": "^4.17.15",
     "memoize-one": "^5.1.1",
     "react": "^16.13.1",
+    "react-async": "^10.0.1",
     "react-dom": "^16.13.1",
     "react-flip-toolkit": "7.0.6",
     "react-helmet": "^5.2.1",
     "react-icons": "^3.10.0",
     "react-redux": "^7.2.0",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "regl": "^1.6.1"
+    "regl": "^1.6.1",
+    "tinyqueue": "^2.0.3"
```

Comment:
What does this get us that array `push`, `pop`, and `length - 1` don't?

Reply:
It is a priority queue, not a simple queue. https://en.m.wikipedia.org/wiki/Priority_queue